examples/pihole/docker-compose.yml

```diff
@@ -10,7 +10,6 @@ services:
         ipv4_address: 172.20.0.2
     dns:
       - 127.0.0.1
-      - 1.1.1.1
```

Comment:
Without this line I see the following errors when `pihole` container starts:

```
...
pihole         |   Checking if custom gravity.db is set in /etc/pihole/pihole-FTL.conf
pihole         |   Skipping Gravity Database Update.
pihole         | 
pihole         | s6-rc: info: service legacy-services successfully started
pihole         | fatal: unable to access 'https://github.com/pi-hole/pi-hole/': Could not resolve host: github.com
pihole         | fatal: unable to access 'https://github.com/pi-hole/AdminLTE/': Could not resolve host: github.com
pihole         | fatal: unable to access 'https://github.com/pi-hole/FTL/': Could not resolve host: github.com
pihole         |   Pi-hole version is v5.15.3 (Latest: null)
pihole         |   AdminLTE version is v5.18.3 (Latest: null)
pihole         |   FTL version is v5.20.1 (Latest: null)
pihole         |   Container tag is: 2023.01.10
```

Then if I `docker exec` into the `pihole` container a `ping` command fails to resolve the name:
```
$ docker exec -it pihole bash
root@pihole:/# ping google.com
ping: google.com: Temporary failure in name resolution
```

Adding back that `1.1.1.1` dns entry the startup looks good:
```
...
pihole         | s6-rc: info: service legacy-services: starting
pihole         |   Checking if custom gravity.db is set in /etc/pihole/pihole-FTL.conf
pihole         |   Skipping Gravity Database Update.
pihole         | 
pihole         | s6-rc: info: service legacy-services successfully started
pihole         |   Pi-hole version is v5.15.3 (Latest: null)
pihole         |   AdminLTE version is v5.18.3 (Latest: null)
pihole         |   FTL version is v5.20.1 (Latest: null)
pihole         |   Container tag is: 2023.01.10
```

Now the ping works:
```
$ docker exec -it pihole bash
root@pihole:/# ping google.com
PING google.com (142.250.200.78) 56(84) bytes of data.
64 bytes from mad07s24-in-f14.1e100.net (142.250.200.78): icmp_seq=1 ttl=249 time=3.66 ms
64 bytes from mad07s24-in-f14.1e100.net (142.250.200.78): icmp_seq=2 ttl=249 time=4.13 ms
...
```